### PR TITLE
[Example]fix argument in cpp inception_bn example

### PR DESCRIPTION
**Pull request by roywei**
## Description ##

- Fix: https://github.com/apache/incubator-mxnet/issues/9417
- Passing correct padding (1, 1) in Pooling in InceptionFactoryB
- Reference [python implementation](https://github.com/apache/incubator-mxnet/blob/master/example/image-classification/symbols/inception-bn.py#L68), `pad=(1, 1)`
- Tested example working fine.

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- Update argument in Pooling

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by anirudhacharya** [comment]
@mxnet-label-bot [pr-awaiting-review, Example]

**Comment by nswamy** [comment]
@roywei can you please fix lint errors?

**Comment by kalyc** [comment]
@mxnet-label-bot update [pr-awaiting-response]

**Comment by roywei** [comment]
@nswamy when adding to tests, I found training acc does not increase. It's happening in other models as well. I have created another issue to track and fix.https://github.com/apache/incubator-mxnet/issues/13243 Most likely root cause will not be relavant to the training scripts here as other models have the same problem.

While this PR can fix the inception-bn example and make it runnable, can we merge it first?

**Comment by roywei** [comment]
@leleamol Could you take a look? Thanks a lot!

**Comment by roywei** [comment]
closed in favor of https://github.com/apache/incubator-mxnet/pull/13284
